cfgov/v1/views/enforcement_api.py

```diff
@@ -86,6 +92,12 @@ class Meta:
         ]
 
 
+@receiver(page_published, sender=EnforcementActionPage)
```

Comment:
Could I suggest possibly locating these signals handlers in the existing [v1/signals.py](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/v1/signals.py) file? We don't have a uniform policy on this, but there's at least some consistency; for example we also have a [jobmanager/signals.py](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jobmanager/signals.py) file.

Per [the Django docs](https://docs.djangoproject.com/en/3.1/topics/signals/#connecting-receiver-functions):

> Where should this code live?
>
> Strictly speaking, signal handling and registration code can live anywhere you like, although it’s recommended to avoid the application’s root module and its models module to minimize side-effects of importing code.
>
> In practice, signal handlers are usually defined in a signals submodule of the application they relate to.

Reply:
When I try to import the `EnforcementActionPage` model into `signals.py`, I get this error: `django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.`. I don't quite understand that one. How can I get around it?

Reply:
Ah. I think that's what's being (unclearly) referenced in the rest of the [Django docs link](https://docs.djangoproject.com/en/3.1/topics/signals/#connecting-receiver-functions) that I didn't include:

> Signal receivers are connected in the ready() method of your application configuration class. If you’re using the receiver() decorator, import the signals submodule inside ready().

Re-reading this, it seems like the signal _handlers_ should be defined in signals.py, but the _connection_ of the signals has to happen in apps.py `ready()` because models aren't available until that time. We do that like this in the v1 app:

https://github.com/cfpb/consumerfinance.gov/blob/f9f3d26b9d054b0f819bd8379dc9278112e98d19/cfgov/v1/apps.py#L15

and slightly differently in the jobmanager app:

https://github.com/cfpb/consumerfinance.gov/blob/f9f3d26b9d054b0f819bd8379dc9278112e98d19/cfgov/jobmanager/apps.py#L9-L11

Reply:
Thanks for your help. Updated in the latest push.